### PR TITLE
Added chocolatey install scripts

### PR DESCRIPTION
**Pull request by rahuliyer95**
#### Description

Chocolatey package generation scripts. Added a few fonts that I use. Rest can be added as described in the README.

#### Requirements / Checklist

- [X] Read the [Contributing Guidelines](https://github.com/ryanoasis/nerd-fonts/blob/master/contributing.md)
- [X] Read or at least glanced at the [FAQ](https://github.com/ryanoasis/nerd-fonts/wiki/FAQ-and-Troubleshooting)
- [X] Read or at least glanced at the [Wiki](https://github.com/ryanoasis/nerd-fonts/wiki)
- [] Scripts execute without error (if necessary):
  - If any of the scripts were modified they have been tested and execute without error, e.g.:
    - `./font-patcher Inconsolata.otf --fontawesome --octicons --pomicons`
    - `./gotta-patch-em-all-font-patcher\!.sh Hermit`
- [X] Extended the README and documentation if necessary, e.g. You added a new font please update the table

#### What does this Pull Request (PR) do?

Adds chocolatey packages and respective generating scripts.

#### How should this be manually tested?

- Run the `generate_packages.py` to generate chocolatey packages for the fonts.
- Go inside any generated folder and run `choco install <font-id> -s .`
    - e.g: `choco install hasklig-nerd-font -s .`
- Fonts should get installed successfully

#### Any background context you can provide?

Chocolatey is a package manager for windows. Easy installation of fonts using the commands like `choco install hasklig-nerd-font`.

#### What are the relevant tickets (if any)?

#223

### CODE REVIEW

**Comment by ryanoasis** [comment]
@rahuliyer95 First of all, great job on this :clap: 

I will have to take some time to review on Windows.

Also just so you know the plan is eventually to segment out this repo. I actually even created a Chocolatey repo here: https://github.com/NerdFonts/Chocolatey

That said I think for now I will merge this in but potentially soon after move the code there and give you contributor rights to the repo.

**Comment by ryanoasis** [comment]
This is too much for me to review all of it so I am just planning on a doing a cursory check and I'll merge this weekend. Thanks again.

I guess the next step is publishing..

**Comment by ryanoasis** [comment]
Hey just wanted to say this is not forgotten. Just have a lot going on right now. This will get merged in eventually. Thanks!!

**Comment by jcjolley** [comment]
Excited to see this :)

**Comment by ryanoasis** [comment]
Wow this really fell off my radar :disappointed: but I'm back now. :smiling_imp:

File: chocolatey/fonts.yml [hunk 1]
```diff
@@ -0,0 +1,111 @@
+version: 2.0.0
+fonts:
+  - name: FiraCode
+    sha256: 09894D24BF3D61493DBA052187A9200497135A4B885CB837BCB637AD2E62070F
+    installed_fonts:
```
Comment (ryanoasis) [inline chocolatey/fonts.yml]: Is there any alternate way to do this? :thinking:

**Comment by ryanoasis** [comment]
Sorry @rahuliyer95 and thanks for your patience. Let's just get this in there so we can iterate on finishing it up...

**Comment by roysubs** [comment]
I see various talk about this in the above, but if I'm reading this correctly, there is no solution? Ideally, I'd like to see either a) a single PowerShell script that can: download latest font, install it, and configure Terminal-Icons (should be simple, I'd be happy to help as I like this project), or b) a Chocolatey package that will do all of the configuration automatically. Has this been done, can someone point me at the complete package or script that does it all? Many thanks.

**Comment by coolaj86** [comment]
@roysubs Is this the sort of thing you're looking for?
https://github.com/webinstall/packages/blob/master/nerdfont/install.ps1 (a la [webinstall.dev/nerdfont](https://webinstall.dev/nerdfont/))

I've got the install complete, but I'm not clear if this accomplishes exactly what you wanted. I'd love to get it to the point that it automatically configures the font in Command.exe (seems not possible) and Windows Terminal (seems possible via JSON, but my powershell-fu is quite weak).

**Pull request merged**